Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN /usr/bin/fix-permissions /tmp/src \
     && /usr/bin/update-ca-trust
 RUN yum install -y krb5-workstation skopeo
 RUN curl -L https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+RUN wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq &&\
```

Comment:
perhaps we can use `https://github.com/houseabsolute/ubi` to maghe tool installations so that its a bit more organized then curl and we get minimal checksum checking?

Also we should probably clamp the version here to make the image a little more reproducible.

Reply:
I eventually plan to remove yq and helm from the tools image altogether (and use a dedicated image for those purposes) so I think what we have so far should be enough for now.

I can mention the exact version instead of latest.

Reply:
I changed installation so it uses a specific version.
If the program decides to go with Helm, then we'll use a dedicated image instead of this one.
If not, then we'd remove Helm and yq from this image.